fsociety.py

```diff
@@ -618,10 +637,19 @@ def __init__(self):
             self.install()
         clearScr()
         print(self.CMSmapLogo)
-        target = raw_input("   Enter a Target: ")
-        self.run(target)
-        response = raw_input(continuePrompt)
-
+        target = raw_input("   Enter a Target: ").split(' ')[0]
+        test_target = ''
+        if target[0:4] == 'http':
+            test_target = target
+        else:
+            test_target = 'http://'+target
+        try:
+            urllib2.urlopen(test_target)
```

Comment:
Hi :smile:,
thanks again for the changes :+1: 

The characters used inside a valid url are also evaluated as valid `bash` directives, which leads anyway to `arbitrary command execution` in minor cases.
The example below is made using the `http://evil.com?||nslookup` PoC payload, and can bypass the protections applied through `urllib2.urlopen()`.

![Screencast-07-25-2020-110717-PM](https://user-images.githubusercontent.com/33063403/88466439-91069780-cecc-11ea-806e-81060ed22af8.gif)
(please excuse the `upper case` when typing ... I hadn't clicked the right key)

I understand it's difficult fixing this type of issue, so I think it's ok using also an external library to validate the `inputs` obtained and avoid `arbitrary command injection` :+1: 

Regards,
Mik